### ISSUE TITLE
Is talking about dicks also misogynistic?

### ISSUE THREAD

**Issue by sklise** [mehan/hip-hop-misogyny-stats#1]
It probably depends on context, but could be interesting to see in comparison to the other words you're tallying.

cc @milliehancock

**Comment by mehan** [comment]
yeah, i did consider that and there were other words i thought about including as well. in the end, i settled on those three terms because their use seemed to be in some way inherently misogynistic, regardless of context. not sure if men talking about their own bodies passes that test? maybe you should fork my repo and build a misandry stats site that only displays lyrics by female rappers?
